### DISCUSSION TITLE
How to play/pause video on single tap

### DISCUSSION THREAD

**Discussion by FXschwartz** [jhomlala/betterplayer#234] [Q&A] [answered]
I know you are able to play/pause when double-tapping the video screen, but how can I make the video pause/play with a single tap?

**Comment by jhomlala** [comment, marked as answer]
Currently there's no possibility. But, you can provide custom controls to player. Check this example:  https://github.com/jhomlala/betterplayer/blob/master/example/lib/pages/change_player_theme_page.dart
https://github.com/jhomlala/betterplayer/blob/master/example/lib/pages/custom_controls/custom_controls_widget.dart#L90
